Source/RapyutaSimulationPlugins/Public/Core/RRActorCommon.h

```diff
@@ -481,7 +481,7 @@ class RAPYUTASIMULATIONPLUGINS_API URRActorCommon : public UObject
     static constexpr const TCHAR* SCRIPT_INI_PATH = TEXT("/Script/RapyutaSimulationPlugins.RRActorCommon");
     static std::once_flag OnceFlag;
 
-    static void OnPostWorldCleanup(UWorld* World, bool bSessionEnded, bool bCleanupResources);
+    static void OnPostWorldCleanup(UWorld* InWorld, bool /*bInSessionEnded*/, bool /*bInCleanupResources*/);
```

Comment:
what is the use case of those anonymouse args?
Could you add as a doxygen documentation?